### PR TITLE
feat: create graphQl Api

### PR DESCRIPTION
**Pull request by AAKlimkov**
1. Move all API to separate modules

### CODE REVIEW

File: src/Api/helpers/queries.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { gql } from "@apollo/client";
+
+const capitalizeFirstLetter = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+
+const generateGetQuery = (queryName: string, fields: string[]) => {
+    const capitalizedTypeName = capitalizeFirstLetter(queryName);
+    return gql`
```
Comment (shev7) [inline src/Api/helpers/queries.ts]: always add space before ```return```

**Review by shev7 — APPROVED** [review]
I tought everything named in lowercase, why api then is named like Api then?

File: src/router/provider.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { RouterProvider } from "react-router-dom";
-import { router } from "./router";
+import { router } from "./router.tsx";
```
Comment (savelievamareena) [inline src/router/provider.tsx]: I think we do not need .tsx extension here for imports to look cleaner

**Review by savelievamareena — APPROVED** [review]

**Review by RC-3222 — APPROVED** [review]

File: src/Api/helpers/queries.ts (hunk 1 above)
Comment (AAKlimkov) [reply src/Api/helpers/queries.ts]: fixed

File: src/router/provider.tsx (hunk 2 above)
Comment (AAKlimkov) [reply src/router/provider.tsx]: fixed

**Review by shev7 — APPROVED** [review]

**Pull request merged**